src/main/java/buchen/bananagrams/Bananagrams.java

```diff
@@ -0,0 +1,56 @@
+package buchen.bananagrams;
+
+import buchen.dictionary.Dictionary;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bananagrams {
+
+    private final Dictionary DICTIONARY;
```

Comment:
Static final variables should be in caps. Member variables should still be in camelcase